docs/ModelStore.md

```diff
@@ -21,6 +21,10 @@ Click on the small download icon (first column) to download a model from the sto
 
 After downloading a model, you can return to the homepage to see the list of pretrained models.
 To make use of the downloaded model for training, select it as a "Pretrained Network" in the "New Model" form.
+Please remember to specify the path of the customized Python layer file if it is required by the downloaded model.
+You are welcome to provide your own Python layer file by uploading it from your browser (Use client-side file).
+Alternatively, you may click "Customize" next to the "Pretrained Model" you select.
+This will automatically populate the path of the Python layer file to the input field of "Python Layers."
```

Comment:
I don't think we need to explain about Python layers in general - we already have a doc for that at `examples/python-layer/`. There's not really any alternative, right? The only way to get the Python layer that comes with the model is to click "Customize". This is very odd and I think deserves some sort of special formatting like:

> *NOTE*: message

(see the note about GPU memory for detectnet [here](https://github.com/NVIDIA/DIGITS/tree/digits-5.0/examples/object-detection#training-detectnet-in-digits))